feature-libs/order/components/order-details/order-overview/order-overview.component.html

```diff
@@ -38,6 +38,12 @@
               ) | async
             "
           ></cx-card>
+
+          <ng-template
```

Comment:
Given that certain some new outlet templates are already enclosed within the *cxFeatureLevel="'6.4'" flag, let's extend this practice to include the recently introduced templates as well. Doing so would enhance the security against unintended template modifications for our customers. 

It's not big deal to remove these feature flags in a follow-up ticket for 7.0.